### PR TITLE
adding `apps` and `overlay` attributes to `flake.nix`

### PR DESCRIPTION
**Pull request by TornaxO7**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -13,17 +13,51 @@
...
+            default = flakeshot;
+
+            flakeshot = {
+              type = "app";
+              program = "${flakeshotPkg}/bin/flakeshot";
+            };
+          });
+
+      overlays.default = final: prev: {
+        flameshot = prev.callPackage mkFlakeshot { };
```
Comment (eneoli) [inline flake.nix:51]: flakeshot

**Review by eneoli — CHANGES_REQUESTED** [review]

File: flake.nix (hunk 1 above)
Comment (TornaxO7) [reply flake.nix]: *ooof*... fixed!

File: flake.nix [hunk 2]
```diff
@@ -13,17 +13,51 @@
...
+        pname = "flakeshot";
+        version = "0.0.1";
+
+        src = ./.;
+        cargoLock.lockFile = ./Cargo.lock;
+
+        cargoSha256 = "";
+
+        meta = {
+          description = "A screenshot tool for wayland and xorg!";
```
Comment (eneoli) [inline flake.nix:29]: xorg or x11?

**Review by eneoli — CHANGES_REQUESTED** [review]

File: flake.nix (hunk 2 above)
Comment (TornaxO7) [reply flake.nix]: fixed => `x11`

**Review by eneoli — APPROVED** [review]

**Pull request merged**